### PR TITLE
Clarifying submit-entry documentation.

### PR DESCRIPTION
**Pull request by eranmes**

### CODE REVIEW

File: draft-ietf-trans-rfc6962-bis.md [hunk 1]
```diff
@@ -1234,9 +1234,10 @@ If the certificate is logged, an SCT MUST be issued. Logging the certificate is
 useful, because monitors ({{monitor}}) can then detect these encoding errors,
 which may be accepted by some TLS clients.
 
-If the returned `sct` is intended to be provided to clients, then `sth` and
+If the returned `sct` is intended to be provided to TLS clients, then `sth` and
 `inclusion` (if returned) SHOULD also be provided to clients (e.g., if `type`
-was 1 then all three `TransItem`s could be embedded in the certificate).
+was `precert_sct_v2` then all three `TransItem`s could be embedded in the
```
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Done.

**Comment by eranmes** [comment]
@robstradling PTAL.

File: draft-ietf-trans-rfc6962-bis.md [hunk 2]
```diff
@@ -1234,9 +1234,10 @@ If the certificate is logged, an SCT MUST be issued. Logging the certificate is
 useful, because monitors ({{monitor}}) can then detect these encoding errors,
 which may be accepted by some TLS clients.
 
-If the returned `sct` is intended to be provided to clients, then `sth` and
+If the returned `sct` is intended to be provided to TLS clients, then `sth` and
 `inclusion` (if returned) SHOULD also be provided to clients (e.g., if `type`
```
Comment (robstradling) [inline draft-ietf-trans-rfc6962-bis.md]: s/clients/TLS clients/

**Comment by robstradling** [comment]
LGTM with one more nit.

File: draft-ietf-trans-rfc6962-bis.md (hunk 2 above)
Comment (eranmes) [inline draft-ietf-trans-rfc6962-bis.md]: Done.

**Pull request merged**